packages/strapi-plugin-i18n/tests/locales.test.e2e.js

```diff
@@ -328,11 +328,23 @@ describe('CRUD locales', () => {
       expect(res.body.message).toBe('Cannot delete the default locale');
     });
 
-    test('Can delete a locale', async () => {
+    test('Can delete a locale - simple', async () => {
+      const res = await rq({
+        url: `/i18n/locales/${data.locales[1].id}`,
+        method: 'DELETE',
+      });
+
+      expect(res.statusCode).toBe(200);
+      expect(res.body).toMatchObject(omitTimestamps(data.locales[1]));
+      data.deletedLocales.push(res.body);
+      data.locales.splice(1, 1);
+    });
+
+    test('Can delete a locale - with related entities', async () => {
```

Comment:
the naming isn't obvious of what you are testing. The test could say sth like "test('Related entities are deleted on locale delete')